Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
def Wurzel(N):
    P = paeckchen(N)

    p = P[0]
    q = quadrat(p)
    r = p - q**2
    i = 1
    l = len(P)

    while (r > 0 or i < l) and i < l + 100:
        if i < l:
            p = P[i]
            r = r * 100 + p
        else:
            r = r * 100

        r, n = rest(r, q)
        q = q * 10 + n
        i += 1
    
    q = q / (10 ** (i - l))

    return q

print(wurzel(17))
print(wurzel(476))
print(wurzel(1097))
print(wurzel(6027341))
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[11], line 25[0m
[1;32m     21[0m     q [38;5;241m=[39m q [38;5;241m/[39m ([38;5;241m10[39m [38;5;241m*[39m[38;5;241m*[39m (i [38;5;241m-[39m l))
[1;32m     23[0m     [38;5;28;01mreturn[39;00m q
[0;32m---> 25[0m [38;5;28mprint[39m([43mwurzel[49m([38;5;241m17[39m))
[1;32m     26[0m [38;5;28mprint[39m(wurzel([38;5;241m476[39m))
[1;32m     27[0m [38;5;28mprint[39m(wurzel([38;5;241m1097[39m))

[0;31mNameError[0m: name 'wurzel' is not defined
NameError: name 'wurzel' is not defined

